src/AppInstallerCommonCore/AppInstallerCommonCore.vcxproj

```diff
@@ -153,7 +153,7 @@
       <PrecompiledHeaderOutputFile>$(IntDir)pch.pch</PrecompiledHeaderOutputFile>
       <PreprocessorDefinitions>_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <WarningLevel>Level4</WarningLevel>
-      <AdditionalOptions>%(AdditionalOptions) /permissive- /D _SILENCE_CXX17_ITERATOR_BASE_CLASS_DEPRECATION_WARNING</AdditionalOptions>
+      <AdditionalOptions>%(AdditionalOptions) /permissive- /D _SILENCE_CXX17_ITERATOR_BASE_CLASS_DEPRECATION_WARNING /D _SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING</AdditionalOptions>

```

Comment:
>_SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING [](start = 120, length = 49)

And apparently you were aware of the deprecation...

Reply:
Yeah, this is the cleanest I can find to convert to utf32, anyway, I used break iterator instead.

---
In reply to: [486835987](https://github.com/microsoft/winget-cli/pull/562#discussion_r486835987) [](ancestors = 486835987)